src/Elmish.WPF/ViewModel.fs

```diff
@@ -381,10 +383,13 @@ and [<AllowNullLiteral>] internal ViewModel<'model, 'msg>
     let dict = Dictionary<string, VmBinding<'model, 'msg>>(bindings.Length)
     let sortedBindings = bindings |> List.sortWith BindingData.subModelSelectedItemLast
     for b in sortedBindings do
-      if dict.ContainsKey b.Name then failwithf "Binding name '%s' is duplicated" b.Name
-      let binding = initializeBinding b.Name b.Data dict
-      dict.Add(b.Name, binding)
-      setInitialError b.Name binding
+      if dict.ContainsKey b.Name
+      then log "Binding name '%s' is duplicated. Only the first occurance will be used." b.Name
```

Comment:
Could you move `then` up to the previous line here?

```f#
if ... then
  log ...
```

Reply:
done